### PR TITLE
feat(*): bump versions for 7.20.0 release

### PR DESCRIPTION
**Pull request by tmetzke**
related to https://github.com/camunda/camunda-bpm-platform/issues/3332

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,16 +27,17 @@
   </modules>
 
   <properties>
-    <version.assertj-core>3.22.0</version.assertj-core>
-    <version.camunda>7.20.0-SNAPSHOT</version.camunda>
+    <version.assertj-core>3.24.2</version.assertj-core>
+    <version.camunda>7.20.0-alpha6</version.camunda>
```
Comment (tmetzke) [inline pom.xml:31]: 🔧 I'll update this to `7.20.0` once all tests have passed in the PR CI.

Comment (tmetzke) [reply pom.xml:31]:
```suggestion
    <version.camunda>7.20.0</version.camunda>
```

**Comment by tmetzke** [comment]
CI successful here with `7.20.0-alpha6`: https://ci.cambpm.camunda.cloud/blue/organizations/jenkins/camunda-github-org%2Fcamunda-archetypes/detail/PR-73/1/pipeline

**Review by danielkelemen — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**